examples/multimodal/multimodal_llm/neva/conf/neva_config.yaml

```diff
@@ -210,6 +210,23 @@ model:
     image_folder: null
     image_aspect_ratio: 'square'
 
+energon:
+  use_energon: False
+  data:
+    __module__: megatron.energon
+    __class__: Metadataset
```

Comment:
can you add a bit more comments here? explain the usage of __class__ and the weights below?